packages/web/package.json

```diff
@@ -39,5 +42,30 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.2.0",
+    "@typescript-eslint/parser": "^3.2.0",
+    "eslint": "^7.2.0",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-plugin-import": "^2.21.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.20.0",
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.10"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn ts:check && yarn test --watchAll=false"
```

Comment:
Can you just omit the watchAll flag?
```
"pre-push": "yarn ts:check && yarn test"
```

Reply:
No. I can't because in the watch mode the task doesn't return code as a result of running `yarn test` that we need to continue to push

Reply:
but is it `true` by default?

Reply:
I think we should keep --watchAll=false, as buy default it is true, what means it will hang checking for updates.

Reply:
Yeah, my bad. That's a weird default of the CRA I wasn't aware of, I'm just used to running `jest --ci` directly.